packages/webpack-config-terra/src/webpack.config.js

```diff
@@ -74,6 +79,20 @@ const getWebpackDevServerStaticOptions = ({ disableHotReloading }) => (
   }
 );
 
+const getAggregateThemeFile = ({ env, themeConfig }) => {
+  const { enableAggregateThemes } = env;
+  if (enableAggregateThemes) {
```

Comment:
We could log out a complain-y message about how you should update your components to not use aggregateThemes because it's super expensive.